controllers/pod_models.go

```diff
@@ -207,6 +219,10 @@ func GetPodSpec(cluster *fdbtypes.FoundationDBCluster, processClass string, idNu
 		return nil, err
 	}
 
+	if processClass == fdbtypes.ProcessClassStorage {
+		sidecarContainer.Env = append(sidecarContainer.Env, corev1.EnvVar{Name: "STORAGE_SERVERS_PER_POD", Value: fmt.Sprintf("%d", cluster.GetStorageServersPerPod())})
```

Comment:
Will this cause a latent change to the environment variables for existing clusters? If so, I think we need to find a way to avoid that.

Reply:
Probably the simplest solution us to set this environment variable only if `storageServersPerPod` is bigger than 1 (which is the default).